commands/apps/destroy.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+let co     = require('co');
+let cli    = require('heroku-cli-util');
+let extend = require('util')._extend;
+let _      = require('lodash');
+
+function* run (context, heroku) {
+  let git = require('../../lib/git')(context);
+
+  let app = context.args.app || context.app;
+  yield heroku.get(`/apps/${app}`);
+  yield cli.confirmApp(app, context.flags.confirm);
```

Comment:
@dickeyxxx The warning about `(including all add-ons)` should be in the confirm, rather than the action.